dashboard/test/models/ability_test.rb

```diff
@@ -797,6 +797,37 @@ class AbilityTest < ActiveSupport::TestCase
     refute Ability.new(student_1).can? :view_as_user_for_code_review, @login_required_script_level, student_2
   end
 
+  test 'only the project owner can create a code review on that project' do
+    project_owner = create :student
+    other_student = create :student
+    code_review = create :code_review, user_id: project_owner.id
+
+    assert Ability.new(project_owner).can? :create, code_review, project_owner.id
+    refute Ability.new(project_owner).can? :create, code_review, other_student.id
```

Comment:
Maybe feels redundant with last refute statement?

Reply:
This is actually a different case.  In this case, the acting user is the owner of `code_review` whereas in the last case the acting user is not the owner of `code_review`.